spras/config/analysis_schema.py

```diff
@@ -0,0 +1,49 @@
+from pydantic import BaseModel, ConfigDict
+
+from spras.config.util_enum import CaseInsensitiveEnum
+
+
+class SummaryAnalysis(BaseModel):
+    include: bool
```

Comment:
Why are these sometimes `bool` and sometimes `bool = True`?

Reply:
The `bool = False` made sense, but I also see `bool = True` and `bool` without a default. Do we intend to have all three?

Reply:
When appropriate, yes :+1: